README.md

```diff
@@ -38,25 +38,33 @@ The actual content of the docs site is found mostly in these folders:
 - `runtime` - docs for the Deno CLI / runtime
 - `deploy` - docs for the Deno Deploy cloud service
 - `subhosting` - docs for Deno Subhosting
+- `by-example` - docs for the [Examples](#Examples) section
 
-Most files are [markdown](https://docusaurus.io/docs/markdown-features), but
-even markdown files are processed with [MDX](https://mdxjs.com/), which enables
-you to use JSX syntax within your markdown files.
+Most files are [markdown](https://lume.land/plugins/markdown/), but even
+markdown files are processed with [MDX](https://mdxjs.com/), which enables you
+to use JSX syntax within your markdown files.
 
-Left navigation for the different doc sections are configured in one of these
-files:
+Left navigation for the different doc sections are configured in the `_data.ts`
+files in their respective content directories.
 
-- `sidebars/runtime.js` - sidebar config for the Runtime section
-- `sidebars/deploy.js` - sidebar config for the Deno Deploy section
-- `sidebars/kv.js` - sidebar config for the KV section
+- `runtime/_data.ts` - sidebar config for the Runtime section
+- `deploy/_data.ts` - sidebar config for the Deno Deploy section
 
 Static files (like screenshots) can be included directly in the `runtime`,
 `deploy`, or `kv` folders, and referenced by relative URLs in your markdown.
 
-Docusaurus provides a number of nice extensions to markdown you might want to
-use, like tabs, admonitions, and code blocks.
-[Refer to the Docusaurus docs](https://docusaurus.io/docs/markdown-features) for
-more details.
+## Reference docs
```

Comment:
just moved this content up next to the section that talks about content locations since it would be relevant to anyone wanting to build the whole docs site.